src/test/rgw/test-rgw-common.sh

```diff
@@ -94,13 +94,11 @@ function init_first_zone {
   x $(rgw_admin $id) realm create --rgw-realm=$realm
 
 # create zonegroup, zone
-  x $(rgw_admin $id) zonegroup create --rgw-zonegroup=$zg --endpoints=http://localhost:$port --master --default
```

Comment:
hi~ @cbodley It seems that we no longer set endpoints for zg,but instead just set endpoints on the master zone of zg? If so, maybe we need to update the doc https://github.com/ceph/ceph/blob/master/doc/radosgw/multisite.rst#creating-a-master-zonegroup